src/huak/ops/fix.rs

```diff
@@ -15,3 +15,52 @@ pub fn fix_project(project: &Project) -> HuakResult<()> {
 
     venv.exec_module(MODULE, &args, &project.root)
 }
+
+#[cfg(test)]
+mod tests {
+    use std::fs;
+
+    use tempfile::tempdir;
+
+    use super::*;
+
+    use crate::utils::{
+        path::copy_dir,
+        test_utils::{create_mock_project, get_resource_dir},
+    };
+
+    #[test]
+    fn fix() {
+        let directory = tempdir().unwrap().into_path().to_path_buf();
+        let mock_project_dir = get_resource_dir().join("mock-project");
+        copy_dir(&mock_project_dir, &directory);
+
+        let project_path = directory.join("mock-project");
+        let project = create_mock_project(project_path.clone()).unwrap();
+        project
+            .venv()
+            .as_ref()
+            .unwrap()
+            .exec_module("pip", &["install", MODULE], &project.root)
+            .unwrap();
+
+        let lint_fix_filepath = project
+            .root
+            .join("src")
+            .join("mock_project")
+            .join("fix_me.py");
+        let pre_fix_str = r##"""
+import json # this gets removed(autofixed)
+
+
+def fn():
+    pass"##;
+
+        fs::write(&lint_fix_filepath, pre_fix_str).unwrap();
+        fix_project(&project).unwrap();
+        let post_fix_str = fs::read_to_string(&lint_fix_filepath).unwrap();
+        println!("{}", post_fix_str);
+
+        assert_ne!(pre_fix_str, post_fix_str);
```

Comment:
Hmm I hate to nit this more than I already have, but maybe we just hardcode the expected `post_fix_str` instead of just checking to make sure it's at least changed.

Also I think you might have left the println for debugging purposes. Can we remove that?